config/interface.cfg

```diff
@@ -48,9 +48,9 @@ fpc_topology_port = 8181
 
 ; The CP Northbound server ip:port: May be configured to any available ip:port
 ; on the CP Host.
-; These values are unused when DSDN_ODL_BUILD CFLAG not defined
-cp_nb_ip = 192.168.125.60
-cp_nb_port = 9997
+; These values are used by CP to fetch CP's IP address and registration port
```

Comment:
Is this the same port where CP waits for incoming connections from DP ? Comment looks bit different

Reply:
Yes. I will make the comment more clear.

Reply:
Done.